### ISSUE TITLE
InnerBlockSlider styles not working

### ISSUE THREAD

**Issue by marcoluzi** [10up/block-components#214]
### Describe the bug

The styles of the InnerBlockSlider seem to be completely off. Default button styles, not positioned correctly, slides are displayed below eachother. Seems to be theme unrelated. I get the same results with my personal theme and also with the [wp-scaffold](https://github.com/10up/wp-scaffold).

### Steps to Reproduce

1. Installing block-components according to instructions.
2. Implementing the InnerBlockSlider according to the example.

### Screenshots, screen recording, code snippet

https://user-images.githubusercontent.com/40238681/229302577-e6a54a4d-07d4-4365-b87b-c8262de477b6.mov

### Environment information

- Device: MacBook Pro 2017
- OS: macOS Monterey 12.6.3
- Browser: Brave v1.49.128

### WordPress information

<details><summary>Site Health Info:</summary>
### wp-core ###

version: 6.1.1
site_language: de_CH_informal
user_language: de_CH_informal
timezone: Europe/Zurich
permalink: /%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 0
default_comment_status: undefined
environment_type: local
user_count: 1
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /Users/marco/repos/valet/lumiere
wordpress_size: 53.91 MB (56533078 bytes)
uploads_path: /Users/marco/repos/valet/lumiere/wp-content/uploads
uploads_size: 2.04 MB (2142666 bytes)
themes_path: /Users/marco/repos/valet/lumiere/wp-content/themes
themes_size: 277.86 MB (291356818 bytes)
plugins_path: /Users/marco/repos/valet/lumiere/wp-content/plugins
plugins_size: 40.08 MB (42022253 bytes)
database_size: 2.00 MB (2097152 bytes)
total_size: 375.89 MB (394151967 bytes)

### wp-active-theme ###

name: Lumiere Theme (lumiere-starter-theme)
version: 3.2.0
author: Marco Luzi
author_website: (undefined)
parent_theme: none
theme_features: widgets-block-editor, automatic-feed-links, title-tag, post-thumbnails, editor-styles, html5, editor-style, menus
theme_path: /Users/marco/repos/valet/lumiere/wp-content/themes/lumiere-starter-theme
auto_update: Deaktiviert

### wp-themes-inactive (3) ###

Twenty Twenty-One: version: 1.7, author: the WordPress team (latest version: 1.8), Automatische Aktualisierungen deaktiviert
Twenty Twenty-Three: version: 1.0, author: WordPress-Team (latest version: 1.1), Automatische Aktualisierungen deaktiviert
Twenty Twenty-Two: version: 1.3, author: WordPress-Team (latest version: 1.4), Automatische Aktualisierungen deaktiviert

### wp-plugins-active (4) ###

Advanced Custom Fields PRO: version: 6.0.7, author: WP Engine, Automatische Aktualisierungen deaktiviert
Loco Translate: version: 2.6.4, author: Tim Whitlock, Automatische Aktualisierungen deaktiviert
Safe SVG: version: 2.0.3, author: 10up (latest version: 2.1.0), Automatische Aktualisierungen deaktiviert
Yoast SEO: version: 20.3, author: Team Yoast (latest version: 20.4), Automatische Aktualisierungen deaktiviert

### wp-plugins-inactive (1) ###

Ghost Kit: version: 2.25.0, author: Ghost Kit Team, Automatische Aktualisierungen deaktiviert

### wp-media ###

image_editor: WP_Image_Editor_GD
imagick_module_version: Nicht verfügbar
imagemagick_version: Nicht verfügbar
imagick_version: Nicht verfügbar
file_uploads: File uploads is turned off
post_max_size: 512M
upload_max_filesize: 512M
max_effective_size: 512 MB
max_file_uploads: 20
gd_version: 2.3.3
gd_formats: GIF, JPEG, PNG, WebP, BMP, AVIF, XPM
ghostscript_version: not available

### wp-server ###

server_architecture: Darwin 21.6.0 x86_64
httpd_software: nginx/1.23.3
php_version: 8.1.17 64bit
php_sapi: fpm-fcgi
max_input_variables: 1000
time_limit: 30
memory_limit: 512M
max_input_time: 60
upload_max_filesize: 512M
php_post_max_size: 512M
curl_version: 8.0.1 (SecureTransport) OpenSSL/1.1.1t
suhosin: false
imagick_availability: false
pretty_permalinks: true

### wp-database ###

extension: mysqli
server_version: 10.11.2-MariaDB
client_version: mysqlnd 8.1.17
max_allowed_packet: 16777216
max_connections: 151

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /Users/marco/repos/valet/lumiere/wp-content
WP_PLUGIN_DIR: /Users/marco/repos/valet/lumiere/wp-content/plugins
WP_MEMORY_LIMIT: 40M
WP_MAX_MEMORY_LIMIT: 512M
WP_DEBUG: true
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: true
SCRIPT_DEBUG: true
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_ENVIRONMENT_TYPE: local
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by marcoluzi** [comment]
I just testet it on a [@10up](https://github.com/10up) / [wp-local-docker-v2](https://github.com/10up/wp-local-docker-v2) setup with https://github.com/10up / [gutenberg-lessons](https://github.com/10up/gutenberg-lessons). Same results.

**Comment by marcoluzi** [comment]
Can I have an update on this? Maybe @fabiankaegy ?

**Comment by marcoluzi** [comment]
@fabiankaegy
Have you given up trying to find the reason for this bug?

**Comment by fabiankaegy** [comment]
Hi @marcoluzi 👋 

Thank you for your report and sorry for the delayed response. The `InnerBlockSlider` was created before core intropduced the apiVersion 2 which now allows us to get rid of any additional wrapper elements in the markup of the editor. Because of that we have shifted to straight up using the same slider libraries as we use on the frontend of the site also in the editor. 

That means the `InnerBlockSlider` has not been maintained in some time. And it seems some of the latest WordPress updates / package dependency updates have caused this regresion here. 

We have created a final fix for this issue in #236 and have added a deprecation warning to the component.
